swarm/pss/pss_test.go

```diff
@@ -371,6 +376,17 @@ func TestHandlerConditions(t *testing.T) {
 	if err := ps.handlePssMsg(fckedupmsg); err == nil {
 		t.Fatalf("expected error from processMsg but error nil")
 	}
+
+	// outbox full should return error
+	msg.Expire = uint32(time.Now().Add(time.Second * 60).Unix())
+	for i := 0; i < defaultOutboxCapacity; i++ {
+		ps.outbox <- msg
+	}
+	msg.Payload.Data = []byte{0x62, 0x61, 0x72}
+	err = ps.handlePssMsg(msg)
+	if err == nil {
```

Comment:
could you not test the actual errors in this test? i mean not just that they are nil or non-nil

Reply:
I guess if we had enumerated messages we could. But for now we don't know which error it hits without parsing the string. Can we leave this for later? I think we should use proper error objects, really.